src/base/static/components/molecules/map-layer-panel-section.css

```diff
@@ -0,0 +1,6 @@
+.map-layer-status-spinner {
+  .react-spinner {
```

Comment:
I wasn't sure how to accomplish this using styled components. The width and height on the spinner have to be set using the `react-spinner` classname.

Reply:
Interesting :thinking:  Are we not able to pass in inline styles to that component, via the `style` prop?

We can consider using Material UI, which I think has much better spinners :)

Reply:
Bah, I totally forgot about the `style` prop, lol. I've been writing so many styled components... :)